tensorflow/cc/framework/testutil.cc

```diff
@@ -36,5 +36,19 @@ void GetTensor(const Scope& scope, Output tensor, Tensor* out) {
   *out = outputs[0];
 }
 
+void GetTensors(const Scope& scope, const std::vector<Output>& assign_vars,
+                OutputList tensors, std::vector<Tensor>* out) {
+  ClientSession session(scope);
+  TF_CHECK_OK(session.Run(assign_vars, nullptr));
```

Comment:
Actually as-is GetTensors instantiates a new ClientSession at each call nonetheless I need to use the same session twice. If you don't want the overload I can instantiate and use a session directly from the test, what do you think?